### DISCUSSION TITLE
Database corrupt with 20221013

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1063] [Fixed Issues]
Artist or AlbumArtist shows only empty Folders with last Build.
Albums are shown.

**Comment by ceejayemm** [comment]
No problem for me here with 20221013 installed.  Are you using the new Shared Data set up ? I am.

Chris

**Comment by hjfisch** [comment]
![Screenshot 2022-10-13 112819](https://user-images.githubusercontent.com/93144100/195559509-27b7845f-636e-4792-882a-b82890fc6acf.jpg)
New Install with latest Build . I've attached an USB_HDD to my RPI 4 and Artist Folder looks like the Screenshot.

**Comment by hjfisch** [comment]
In my USB-Folder everyting is shown
![Screenshot 2](https://user-images.githubusercontent.com/93144100/195561320-a9e5b361-b213-4072-aea7-ba97c8680f20.jpg)

**Comment by rern** [comment]
Fixed by update 202210131.
(Update Library database again to restore the incomplete items.)

**Reply by hjfisch** [reply]
Thank's , working now
